google/cloud/bigtable/backup.py

```diff
@@ -0,0 +1,360 @@
+# Copyright 2015 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""User friendly container for Cloud Bigtable Backup."""
+
+import re
+
+# import warnings
+# from datetime import datetime
+
+from google.cloud._helpers import _datetime_to_pb_timestamp
+from google.cloud.bigtable_admin_v2.gapic import enums
+
+# from google.cloud.bigtable_admin_v2.proto import table_pb2
+# from google.cloud.bigtable_admin_v2.proto import bigtable_table_admin_pb2
+from google.cloud.exceptions import NotFound
+
+_BACKUP_NAME_RE = re.compile(
```

Comment:
`re` is a part of Python standard library. Since it is used only once, I moved it into the `from_pb` method, to make testing more explicit. The coverage shows 100% either way.